utils/simpleusb-tune-menu.c

```diff
@@ -774,7 +785,7 @@ static int astgetresp(char *cmd)
 		}
 
 		printf("1) Select active USB device\n");
-		printf("2) Set Rx Voice Level (using display)\n");
+		printf("2) Set Rx Voice Level w/display (currently '%d')\n", rxmixerset);
```

Comment:
Allen how do you feel about replacing 'w/display' with 'with display'?

Reply:
Done.